code/modules/antagonists/bloodsucker/powers/lunge.dm

```diff
@@ -7,9 +7,9 @@
 	button_icon_state = "power_lunge"
 	bloodcost = 10
 	cooldown = 120
-	target_range = 3
+	target_range = 5
```

Comment:
Iffy on nearly doubling its range.

Reply:
they don't move any faster and with telegraphing further ranges gives more time to react